lib/game/client/manager.js

```diff
@@ -332,6 +334,15 @@ class GameManager
         });
     }
 
+    getGameEngine()
+    {
+        if (!this.gameEngine)
```

Comment:
remove the shortcut, this.gameEngine should be check on creation and if not present throw an error

you can even add a validate() method that will check all the, this.engine, this.bla, this.bablbalba, etc. at once and throw an error if any is missing